Modules/Office365DSC/DSCResources/MSFT_TeamsGuestSettings/MSFT_TeamsGuestSettings.schema.mof

```diff
@@ -0,0 +1,10 @@
+[ClassVersion("1.0.0.0"), FriendlyName("TeamsGuestSettings")]
+class MSFT_TeamsGuestSettings : OMI_BaseResource
+{
+    [Key, Description("The Name of the team")] string TeamName;
+    [Write, Description("Allow guests to create and update channels in team ")] Boolean AllowCreateUpdateChannels;
+    [Write, Description("Allow guests to delete channel in team ")] Boolean AllowDeleteChannels;
+    [Write, Description("Placeholder only not used in resource."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Required, Description("Credentials of the SharePoint Global Admin"), EmbeddedInstance("MSFT_Credential")] string GlobalAdminAccount;
```

Comment:
Description should probably be Credentials of the Office 365 Tenant Admin

Reply:
done